manifests/opensearch-1.1.0.0.yml

```diff
@@ -0,0 +1,51 @@
+---
+schema-version: 1.0
+build:
+  name: OpenSearch
+  version: 1.1.0
+components:
+  - name: OpenSearch
+    repository: https://github.com/opensearch-project/OpenSearch.git
+    ref: 1.x
+  - name: common-utils
+    repository: https://github.com/opensearch-project/common-utils.git
+    ref: main
+  - name: job-scheduler
+    repository: https://github.com/dblock/job-scheduler.git
+    ref: main
+  - name: sql
+    repository: https://github.com/opensearch-project/sql.git
+    ref: main
+  - name: alerting
+    repository: https://github.com/dblock/alerting.git
```

Comment:
Can you pls update this to not point to your forks.

Reply:
This is on my forks because the fix PRs haven't been merged yet. Hence this whole thing is in draft.